src/crdt.rs

```diff
@@ -551,6 +551,33 @@ impl Crdt {
         blob_sender.send(q)?;
         Ok(())
     }
+    /// FIXME: This is obviously the wrong way to do this. Need to implement leader selection
```

Comment:
Let's consistently used TODO or FIXME. Codebase currently uses TODO. If you prefer FIXME, can you change the others?

Reply:
I thought FIXME was a higher priority todo :)